universal-login-commons/test/core/utils/bigNumbers.ts

```diff
@@ -0,0 +1,17 @@
+import chai, {expect} from 'chai';
+import {normalizeBigNumber} from '../../../lib/core/utils/bigNumbers';
+import {BigNumber} from 'ethers/utils';
+import {solidity} from 'ethereum-waffle';
+
+chai.use(solidity);
+
+describe('UNIT: BigNumber', () => {
+  it('0x00', () => {
+    const a = new BigNumber('0x00');
+    const b = new BigNumber('0x0');
+
+    const expected = normalizeBigNumber(b);
+
+    expect(expected).to.be.equal(a);
```

Comment:
A couple more tests wouldn't hurt.

Reply:
As we know now, we should probably not use `expect(...).to.be.equal(...)` as that is a bit insidious for `BigNumber` type.